helm-chart/renku-notebooks/templates/deployment.yaml

```diff
@@ -51,6 +51,8 @@ spec:
               value: {{ .Values.jupyterhub.hub.baseUrl }}services/notebooks/
             - name: JUPYTERHUB_BASE_URL
               value: {{ .Values.jupyterhub.hub.baseUrl }}
+            - name: JUPYTERHUB_ORIGIN
+              value: {{ template "notebooks.http" . }}://{{ .Values.global.renku.domain }}
```

Comment:
should the default be `.Values.jupyterhub.url`?

Reply:
In most of our charts it's the other way around: a global chart value is the default which can be overridden by a local one. However, I would prefer not to do this here. In fact I don't see a need for this as a standard pattern and I would like to move away from it. It just results in even more places where the same deployment parameter can be set. Either a value is the same across all renku services and should thus be global, or it's something that should be set for each service specifically and thus local. Or am I missing something here?

Reply:
ok but the default values file for the notebooks service doesn't define `renku.domain` in its `global` section

Reply:
But in this case the `JUPYTERHUB_API_URL` just a few lines above will also be undefined. I tried get away with minimal chart changes for this one and use the values which are already there. I agree there's more to be fixed, but let's try to keep PRs small.